engine/Shopware/Controllers/Backend/Log.php

```diff
@@ -188,15 +188,20 @@ public function getLogFileListAction()
         $files = $this->getLogFiles($logDir);
         $defaultFile = $this->getDefaultLogFile($files);
 
-        $files = array_map(function ($file) use ($defaultFile) {
-            return [
+        // filter against input
+        $query = trim($this->Request()->getParam('query', ''));
+        foreach ($files as $k => $file) {
+            if (!empty($query) && mb_stripos($file[0], $query) === false) {
```

Comment:
```suggestion
            if ($query !== '' && mb_stripos($file[0], $query) === false) {
```
I would check against an emptry string, so it is more explicit. `empty` checks to much. e.g. the string `'0'` will evaluate to empty and this is not intended, is it?